classes/Entities/Cohort.php

```diff
@@ -5,4 +5,9 @@
 class Cohort extends BaseModel
 {
     protected $table = 'cohorts';
+
+    Public function members()
```

Comment:
I guess this 'Public' should be lowercase. 
also Andrew asked me to just add basic docs to all my variables, classes and functions